handlers/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
-
 - name: restart nessusagent
   service: name=nessusagent state=restarted
+
+- name: restart nessus_link
+  systemd: name=nessus_link state=restarted daemon_reload=true
```

Comment:
Hey, We are currently using this role with ubuntu 14.04 and intend to have this role continue to work with versions prior to 16.04. In order to consider this PR we would need it to support the current method and only use the systemd method for systems that use it as the default service manager.